code/modules/power/apc/apc_main.dm

```diff
@@ -451,6 +451,9 @@
 		main_status = APC_NO_POWER
 	else if(excess < 0)
 		main_status = APC_LOW_POWER
+		for(var/obj/machinery/light/flickering_light in area)
+			flickering_light.flickering = TRUE // We don't need to unset, because it resets back to FALSE every time it flickers in light code
+			CHECK_TICK
```

Comment:
This seems very wrong, especially since it'll delay stuff like the actual power cell usage